### ISSUE TITLE
New concept:  Anomalous Cosmic Rays

### ISSUE THREAD

**Issue by ace91024** [astrothesaurus/UAT#447]
**Name the concept in question**
Which concept is this suggestion about? Please provide the numeric ID or URI for the concept as well.

**Describe the change**
Should the concept be moved to another location within the hierarchy? Where and why?
Should the concept be renamed? Why?
Should this concept be merged with another concept? Which concept and why?

**Please include any additional comments/feedback**

**Comment by ace91024** [comment]
how do I logout of this website?

**Comment by BartlettAstro** [comment]
@ace91024 you are correct, not all energetic particles originate in the Sun. With what kind of energetic particles are you working?  We need a little more context to help you identify an appropriate existing concept or add a concept that should be included but is not already.

**Comment by BartlettAstro** [comment]
"Solar energetic particles" is UAT concept 1491

**Comment by ace91024** [comment]
Hi:

We are studying Anomalous Cosmic Rays.

Best,
Alan

**Comment by BartlettAstro** [comment]
### Proposal for discussion

Preferred Label:  Anomalous cosmic rays

Broader concepts:  Cosmic rays (https://astrothesaurus.org/uat/329)

Narrower concepts: None

Related concepts: 
Cosmic ray astronomy (https://astrothesaurus.org/uat/324)
Interstellar medium (https://astrothesaurus.org/uat/847)
Solar wind (https://astrothesaurus.org/uat/1534)
Heliosphere (https://astrothesaurus.org/uat/711)
Termination shock (https://astrothesaurus.org/uat/1690) 

Alternate terms: none

Definition:
Anomalous cosmic rays have energies of ~107 to 108 eV and are thought to originate from neutral interstellar atoms that have been ionized by solar UV radiation after entering the heliosphere. Once ionized, these charged particles are carried on the solar wind and associated magnetic field back out to the shock created where the solar wind meets interstellar space. This shock accelerates the nuclei to cosmic ray energies, some of which then return to the inner Solar System.
COSMOS - The SAO Encyclopedia of Astronomy (https://astronomy.swin.edu.au/cosmos/*/Anomalous+Cosmic+Rays)


@ace91024 would adding anomalous cosmic rays meet your needs?
@rmcgranaghan do you have any thoughts on adding this concept?

**Comment by msdemlei** [comment]
On Mon, May 19, 2025 at 01:06:56PM -0700, Jennifer Lynn Bartlett wrote:
> Anomalous cosmic rays have energies of ~107 to 108 eV and are

Let's either write "1e7 to 1e8 eV" or (my favourite) "10 to 100 MeV"
or, if you want to be fancy, "10⁷ to 10⁸ eV".  We can't have
superscript markup in our descriptions.

> thought to originate from neutral interstellar atoms that have been
> ionized by solar UV radiation after entering the heliosphere. Once
> ionized, these charged particles are carried on the solar wind and
> associated magnetic field back out to the shock created where the
> solar wind meets interstellar space. This shock accelerates the
> nuclei to cosmic ray energies, some of which then return to the
> inner Solar System.
> COSMOS - The SAO Encyclopedia of Astronomy (https://astronomy.swin.edu.au/cosmos/*/Anomalous+Cosmic+Rays)

I'm still skeptical of these attributions, partly because it's
questionable if we can still have them when we do editorial changes
like the one proposed above.

But then I don't much like this particular definition anyway.  You
see, the definition should delineate the extension of the concept.
In this case, the extension would seem to be the set of things
"thought to be <whatever>".  I'd much better like it if we said what
someone *is* rather than what it is *thought of*.  Perhaps:

  Anomalous Cosmic Rays is particle radiation within the Solar
  System of energies much lower than 1 GeV (typcially some 10s of
  MeV).  Galactic cosmic rays in that energy range should not pass
  the heliosphere, which is why these particles were called anomalous
  when they were discovered.  They are now understood to be ionised
  interstellar atoms accelerated in the outer Heliosphere.

I notice in passing that
https://link.springer.com/article/10.1007/s11214-022-00890-7 proposes
"Heliospheric Energetic Particles" as  modern label for that concept,
and I think I'd like that better as the preferred label.  With that,
perhaps the last sentence of my proposed definition could become the
first.
